mackup/applications/forklift-3.cfg

```diff
@@ -0,0 +1,5 @@
+[application]
+name = ForkLift 3
+
+[configuration_files]
+Library/Preferences/com.binarynights.ForkLift-3.plist
```

Comment:
I'm sorry, I forgot that there was a native sync feature. I will remove this change request.